<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fourthDialog</class>
 <widget class="QDialog" name="fourthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fourthDialog{
border-image : url(:/admin_panel.png);
}</string>
  </property>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font: 18pt &quot;century gothic&quot;;
font-size:28px;
color:white;
background: #000000;
}
QLabel{
font: 18pt &quot;century gothic&quot;;
font-size:28px;
background:#ffffff;

}
QScrollArea QWidget
{  font: 18pt &quot;century gothic&quot;;
font-size:28px;

    background-color:rgb(0, 85, 127);
}


</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>339</width>
      <height>259</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton{
color:rgb(255, 255, 255);
background-color: rgb(0, 85, 127);
	font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#pushButton:hover{
background-color:rgb(0, 100, 147);
font: 75 14pt &quot;Sitka Display&quot;;

}

</string>
   </property>
   <property name="text">
    <string>Sign out</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>110</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit{
font: 75 14pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:1px;

}
</string>
   </property>
   <property name="placeholderText">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2{
	
background-color: rgb(255, 255, 255);
	font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#pushButton_2:hover{
background-color:rgb(115, 255, 105);
font: 75 14pt &quot;Sitka Display&quot;;

}

</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#00557f;&quot;&gt;Enter Title Name :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_2{
border-width:0px;
border-radius:1px;
font: 20px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Search Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3{
color:rgb(255, 255, 255);
	font: 75 18pt &quot;Sitka Display&quot;;

background-color: rgb(0, 85, 127);
border-width:0px;
border-radius:1px;
}
#pushButton_3::hover{
font: 75 14pt &quot;Sitka Display&quot;;

background-color:rgb(0, 111, 163);

}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{font: 18pt &quot;century gothic&quot;;
font-size:28px;
color:white;
background: #000000;
}
QLabel{
font: 18pt &quot;century gothic&quot;;
font-size:28px;
background:#ffffff;

}
QScrollArea QWidget
{  font: 18pt &quot;century gothic&quot;;
font-size:28px;

    background-color:rgb(0, 85, 127);
}

</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>689</width>
      <height>469</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>981</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>481</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_8{
background-color:rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-1</y>
       <width>751</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_4{
color:rgb(255, 255, 255);
	
background-color: rgb(0, 85, 127);
	font: 75 16pt &quot;Sitka Display&quot;;
border-width:0px;
border-radius:4px;
}
#pushButton_4:hover{
background-color:rgb(0, 96, 141);
font: 75 14pt &quot;Sitka Display&quot;;

}

</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>660</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;© 2019 Team Noob.  All rights reserved.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
